scanner/cmd/updater/main.go

```diff
@@ -16,8 +16,7 @@ import (
 const DefaultURL = "https://raw.githubusercontent.com/stackrox/stackrox/master/scanner/updater/manual/vulns.yaml"
 
 func tryExport(ctx context.Context, outputDir string, opts *updater.ExportOptions) error {
-	// Temporarily increase update timeout for Vex update
-	const timeout = 6 * time.Hour
+	const timeout = 3 * time.Hour
```

Comment:
Is there any harm in leaving this at 6h (the GH max)?

Reply:
Technically, this could build up a queue of the updater GHA jobs, which we briefly observed over the weekend. I think it's unlikely to cause a major problem, though. 5h seems like a safe value.